rotkehlchen/exchanges/kraken.py

```diff
@@ -699,7 +699,7 @@ def query_online_trade_history(
             location=Location.KRAKEN,
             location_label=self.name,
         )
-        trades_raw = self.db.get_history_events(filter_query)
+        trades_raw, _ = self.db.get_history_events(filter_query)
```

Comment:
This is exactly why for others we have `get_xxx()` and `get_xxx_and_limit_info()`. So that you don't have to go and do those changes. Please use the already existing logic.